content/08-mongodb/05-authorization-and-privileges.mdx

```diff
@@ -1,7 +1,7 @@
 ---
-title: 'How to manage authorization and privileges in MongoDB'
+title: 'MongoDB Role Management, Authorization, and Privileges'
```

Comment:
If we're changing the actual title here, it needs to be updated in the index file as well.

Reply:
hello @imchairmanm - apologies for the delay here, didn't see that this was kicked back to me.

Great catch - I had meant to update the title tag, not the H1. I reverted that field back to the original, and applied the desired changes to the meta title as intended.

Thanks!